doc/mgmt/SONiC_OC_Interface_HLD.md

```diff
@@ -392,6 +396,7 @@ sonic(conf-if-po4)# no switchport access Vlan
    mode - active<br>
    min-links - 0<br>
    fallback - disabled<br>
+   fast_rate - disabled<br>
```

Comment:
Fast_rate is to configure the rate at which control packets are sent by the Link Aggregation Control Protocol (LACP) ??
If we enable the fast_rate config, what is the default rate at which lacp PDUs will be sent?
What are the configurable rates?
is there a no form of this command ??

Reply:
LACP has only 2 rates - slow (once in 30 seconds) and fast (once in 1 sec).

This is a parameter that cannot be changed once the portchannel is created. So, the only way to change it is to delete and recreate the portchannel.